README.md

```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
 
 # &#127939; How do I get started?
 
-## Embedding pipeline
+## Data Scraping pipeline
+
+Before we start running the pipeline, an additional setup for creating a storage container on Azure is required to store the data. We use [DVC](https://dvc.org/doc/user-guide/data-management/remote-storage/azure-blob-storage) tool for data versioning and data management.
+
+> Data version [documentation](docs/data-version-control.md) guides you through the process of setting up a storage container on Azure and configuring DVC.
```

Comment:
If the user needs to go and read this as part of the setup, then it should be included in the paragraph above.